app.js

```diff
@@ -74,6 +91,9 @@ async function recoveryMethodsFor(account) {
     });
 }
 
+/********************************
```

Comment:
seems like need a rebase after my changes here https://github.com/near/near-contract-helper/pull/188

Reply:
I keep getting into a rebase loop here, maybe you can help me or have a different strategy.